### ISSUE TITLE
Vagrant up failing on drush install step (Drupal 7 environment)

### ISSUE THREAD

**Issue by adamtyoung** [geerlingguy/drupal-vm#1670]
## Issue Type

 - Bug Report / Support Request

### Your Environment

```
Vagrant 2.0.1
VirtualBox 5.2.6r120293
ansible 2.4.2.0
  config file = None
  configured module search path = [u'/Users/adamtyoung/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /Library/Python/2.7/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 2.7.10 (default, Jul 15 2017, 17:16:57) [GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.31)]
```

### Your OS

 - **macOS High Sierra** 

### Full console output

https://gist.github.com/adamtyoung/1aa52d4f0a4ec88f72d527c45a1e64bb

### Summary

Here's my latest issue with DVM--it keeps trying download Drush 7 .phar files even though those were never packaged (cf this 'closed' issue https://github.com/geerlingguy/drupal-vm/issues/1235). I have explicitly put ```drush_version: "8.1.15"``` in my config.yml file.

**Comment by geerlingguy** [comment]
@adamtyoung - Are you sure that `drush_version` is set to `8.1.15` in config.yml? Somewhere it's picking up a `7.x`; are there any other instances of `7.x` in your config.yml or in default.config.yml? Also, are you customizing any of the other Drush role variables?

Finally, I just pushed an update to Drupal VM `master` for the Drush role itself, though it doesn't seem like the fixes in there would have any relation to the version of the phar downloaded.

**Comment by adamtyoung** [comment]
Yes you are right---I tracked down another entry for drush v 7---now I am having the problem that I had before of the Remi repo failing:

https://gist.github.com/adamtyoung/ea521f53a936c2d39caf7a91a0be8376

Are these somehow related?

**Comment by geerlingguy** [comment]
No, that looks like something else entirely.

I'm going to close this ticket since it seems like the original issue is solved, and if you still have issues with the Remi repo you can open a new ticket, but first try:

* Load http://rpms.remirepo.net/enterprise/remi-release-7.rpm in your browser and see if it downloads the file. If not, maybe your network is blocking it or something? (Are you using a proxy?).
* Log into the VM (`vagrant ssh`) and run `curl http://rpms.remirepo.net/enterprise/remi-release-7.rpm` and see what that gives you.

Basically it looks like something is blocking access to that URL... or maybe it was just temporary and if you run `vagrant provision` again it will just start working.
